grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -637,7 +637,10 @@ public GrpcServiceBuilder unframedGrpcErrorHandler(UnframedGrpcErrorHandler unfr
     /**
      * Sets whether the service handles HTTP/JSON requests using the gRPC wire protocol.
      *
-     * <p>Limitations:
+     * <p><b>Limitations:</b>
+     * Make sure {@code GrpcSerializationFormats.JSON} is supported by the service by calling
+     * {@code this#supportedSerializationFormats(GrpcSerializationFormats.JSON)}. Otherwise, service builder
```

Comment:
How about adding this sentence at last instead of containing it in this section?
```
     * <p>{@link GrpcSerializationFormats#JSON} must be supported via
     * {@link #supportedSerializationFormats(SerializationFormat...)} to enable HTTP/JSON transcoding.
```